### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by grahamc**
I don't write much Go, but it works.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+set -eux
+
+PROJECT_NAME=bonk-api
+
+nix build .#packages.x86_64-linux.dockerImage
+skopeo \
+    --insecure-policy \
+    copy docker-archive:"$(realpath ./result)" \
+    docker://registry.fly.io/$PROJECT_NAME:latest \
+    --dest-creds x:"$(flyctl auth token)" \
```
Comment (lheckemann) [inline deploy.sh:13]: noooooooooo

Comment (grahamc) [reply deploy.sh:13]: please send alternatives :sweat:

Comment (lheckemann) [reply deploy.sh:13]: `skopeo login` accepts a `--password-stdin` option, so
```
skopeo login docker://registry.fly.io --username x --password-stdin < <(flyctl auth token)
```
then this without `--dest-creds` should hopefully work

**Review by cole-h — APPROVED** [review]
LGTM

File: deploy.sh (hunk 1 above)
Comment (grahamc) [reply deploy.sh:13]: Will fix in a follow-up shortly. Thanks!

**Pull request merged**